integration-tests/create-react-app/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "create-react-app",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "react": "^15.6.1",
+    "react-dom": "^15.6.1"
+  },
+  "optionalDependencies": {
+    "react-scripts": "1.0.11"
```

Comment:
Oh I see there were more changes here — I'm pretty comfortable with Node 6+ since that is the most recent LTS  version. Depends I guess on how hard it would be to guarantee lower version compatibility.